Dockerfile

```diff
@@ -38,3 +38,10 @@ ENV XDG_CACHE_HOME /tmp/.cache
 EXPOSE 8089
 ENV PORT 8089
 CMD uwsgi uwsgi.ini
+
+# Second stage build installs reqs needed only for develoment envs
+# Invoke 'requirements.txt' again because 'test_requirements.txt' doesn't 
+# like to run alone for some reason
+FROM production AS development
+COPY test_requirements.txt /tmp/test_requirements.txt
+RUN pip install -r requirements.txt -r test_requirements.txt
```

Comment:
Should be:
```
RUN pip install -r requirements.txt -r /tmp/test_requirements.txt
```
to match the location of the COPY command above.  Otherwise the test requirements aren't installed:

```
odl-video-service % docker-compose run web scripts/test/python_tests.sh

Starting odl-video-service_redis_1 ... done
Starting odl-video-service_db_1    ... done
Creating odl-video-service_web_run ... done
[TEST SUITE] black --check .
scripts/test/python_tests.sh: line 12: black: command not found

[TEST SUITE] pytest
scripts/test/python_tests.sh: line 12: pytest: command not found
```